app/_hub/kong-inc/request-transformer/index.md

```diff
@@ -3,9 +3,9 @@ name: Request Transformer
 publisher: Kong Inc.
 version: 1.2.5
 
-desc: Modify the request before hitting the upstream server
+desc: Use regular expressions, variables and templates to transform requests
 description: |
-  Transform the request sent by a client on the fly on Kong, before hitting the upstream server.
+  The Request Transformer plugin for Kong allows simple transforming the requests before they reach the upstream server. These transformations can be simple substitutions or complex ones matching portions of incoming requests using regular expressions, save those matched strings into variables, and substitute those strings into transformed requests via flexible templates.
```

Comment:
```suggestion
  The Request Transformer plugin for Kong allows simple transformation of requests before they reach the Upstream server. These transformations can be simple substitutions or complex ones matching portions of incoming requests using regular expressions, saving those matched strings into variables, and substituting those strings into transformed requests using flexible templates.
```

Reply:
Took all the suggestions here except "Upstream server" - in this case we are using "upstream" as an adjective. And yes, this is more confusing than it needs to be. I personally think we should have named our Upstreams differently.